modin/data_management/functions/default_methods/any_default.py

```diff
@@ -26,7 +26,20 @@ class ObjTypeDeterminer:
     to an object under which `key` function is applied.
     """
 
-    def __getattr__(self, key):  # noqa: D105
+    def __getattr__(self, key):
+        """
+        Build function that executes `key` function under passed frame.
```

Comment:
```suggestion
        Build function that executes `key` function over passed frame.
```
or against, pick one :)